### PR TITLE
Misc bug and release testing

### PR DESCRIPTION
**Pull request by rocky**

### CODE REVIEW

File: mathics_django/web/format.py [hunk 1]
```diff
@@ -43,8 +46,6 @@ def eval_boxes(result, fn, obj, **options):
     if isinstance(format, dict):
         return dict((k, obj.format_output(expr, f)) for k, f in format.items())
 
-    from mathics.core.expression import Expression, BoxError
```
Comment (TiagoCavalcanteTrindade) [inline mathics_django/web/format.py:46]: Why remove it from here? Isn't lazy reload good?

File: mathics_django/web/format.py [hunk 2]
```diff
@@ -65,9 +66,7 @@ def eval_boxes(result, fn, obj, **options):
 
     # This part was derived from and the same as evaluation.py format_output.
 
-    use_quotes = (
-        Expression("Settings`$QuotedStrings").evaluate(obj).get_head().to_python()
-    )
+    use_quotes = get_settings_value(obj.definitions, "Settings`$QuotedStrings")
```
Comment (TiagoCavalcanteTrindade) [inline mathics_django/web/format.py:69]: Great!

File: mathics_django/web/format.py (hunk 1 above)
Comment (rocky) [reply mathics_django/web/format.py:46]: You spend _way_ too much time worrying about this kind of stuff

Expression is needed to format just about everything. 

But tell you what - run some tests to see how much time you save doing this. I doubt you will find a difference. But prove me wrong!

Comment (TiagoCavalcanteTrindade) [reply mathics_django/web/format.py:46]: > You spend way too much time worrying about this kind of stuff

Ok, I'll try to stop doing this.

**Pull request merged**